fem/ceed/integrator.hpp

```diff
@@ -119,17 +144,38 @@ class PAIntegrator : public ceed::Operator
                      info.test_op
                     };
       CeedInt nqpts, nelem = mesh.GetNE();
-      CeedInt dim = mesh.SpaceDimension(), vdim = fes.GetVDim();
+      CeedInt dim = mesh.SpaceDimension();
+      CeedInt trial_vdim = trial_fes.GetVDim();
+      CeedInt test_vdim = test_fes.GetVDim();
 
       mesh.EnsureNodes();
-      InitBasisAndRestriction(fes, irm, ceed, &basis, &restr);
+      if ( &trial_fes == &test_fes )
```

Comment:
I guess the FES object doesn't have an operator== overload defined.

Reply:
I think using pointer comparison in this context is okay, but I agree that using `==` would be safer, cleaner, and more generic.